### DISCUSSION TITLE
FreeTransform: Resizing of the element works incorrectly when user resizes browser window

### DISCUSSION THREAD

**Discussion by gintarasgraciovas** [clientIO/joint#2590] [Q&A]
### Introduction

If the user enters the page with the browser window at 75% width, then adds an `element` from the stencil, resizes the screen to 25%, and attempts to resize an element on the graph, at some point it resizes it to the smallest size.

This is my code:
![image](https://github.com/clientIO/joint/assets/146833191/ba55ccf9-5655-4854-8a85-e4c920c53bec)

![image](https://github.com/clientIO/joint/assets/146833191/d73a5b1f-7705-4729-ac3d-6d7c690c8721)

### Steps to reproduce

1) Enter the page with the browser window at 75% width;
2) Add an element from the stencil;
3) Resize the screen to 25%;
4) Try to resize an element on the graph and observe that at some point it resizes it to the smallest size

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS, JointJS+

**Comment by kumilingus** [comment]
I can't reproduce.

Are you able to reproduce the issue with one of our online demos?
Could you perhaps record a video of the issue? Thank you.

> Resize the screen to 25%

Do you mean to change the browser's zoom level?

**Reply by gintarasgraciovas** [reply]
Thanks for the reply, I doubt that I'd be able to reproduce it in any of your demos..
Here is the the screen recording, hopefully it will help:

https://github.com/clientIO/joint/assets/146833191/2bd14dd1-5fab-4c7b-accd-3c68107fe875

> Do you mean to change the browser's zoom level?

Yes, I mean browser's zoom.

**Reply by kumilingus** [reply]
Does the error happen in a specific browser?

If only reproducible in Safari it could be related to this [bug](https://bugs.webkit.org/show_bug.cgi?id=257090).
It seems that `getScreenCTM()` returns a wrong matrix.

**Reply by gintarasgraciovas** [reply]
> Does the error happen in a specific browser?

> If only reproducible in Safari it could be related to this [bug](https://bugs.webkit.org/show_bug.cgi?id=257090).
> It seems that getScreenCTM() returns a wrong matrix.

It's actually occurring in the Google Chrome browser. I've just realized that there might be another issue at play. Essentially, when I drop an asset onto the graph, it seems to appear at different coordinates. I'll investigate it and let you know..
